docker-compose.yml

```diff
@@ -11,13 +11,13 @@ services:
     networks:
       - default
   pattern-pedia-api:
-    image:  patternpedia/patternrepo-api:latest
+    image:  testapi
```

Comment:
I guess this is only valid vor testing purposes, please revert.

Shouldn't we maybe switch to another organization in DockerHub? Just to align it with the orga name in GitHub....